src/EventHandler/Message/Service/DialogCallGroup/Called.php

```diff
@@ -0,0 +1,41 @@
+<?php declare(strict_types=1);
+
+/**
+ * This file is part of MadelineProto.
+ * MadelineProto is free software: you can redistribute it and/or modify it under the terms of the GNU Affero General Public License as published by the Free Software Foundation, either version 3 of the License, or (at your option) any later version.
+ * MadelineProto is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+ * See the GNU Affero General Public License for more details.
+ * You should have received a copy of the GNU General Public License along with MadelineProto.
+ * If not, see <http://www.gnu.org/licenses/>.
+ *
+ * @author    Daniil Gentili <daniil@daniil.it>
+ * @copyright 2016-2023 Daniil Gentili <daniil@daniil.it>
+ * @license   https://opensource.org/licenses/AGPL-3.0 AGPLv3
+ * @link https://docs.madelineproto.xyz MadelineProto documentation
+ */
+
+namespace danog\MadelineProto\EventHandler\Message\Service\DialogCallGroup;
+
+use danog\MadelineProto\EventHandler\Message\Service\AbstractDialogCallGroup;
+use danog\MadelineProto\MTProto;
+
+/**
+ * The group call has ended.
+ */
+final class Called extends AbstractDialogCallGroup
```

Comment:
Can we name the namespace `DialogGroupCall`?
Also, can we just name the class `GroupCallEnded`? I don't like using such non-descriptive names for the class (even if more info is included in the namespace).

Reply:
Same for the other classes in the DialogGroupCall namespace